src/openrct2/ride/Ride.cpp

```diff
@@ -2078,8 +2078,12 @@ void Ride::Update()
 
     // Various things include news messages
     if (lifecycle_flags & (RIDE_LIFECYCLE_BREAKDOWN_PENDING | RIDE_LIFECYCLE_BROKEN_DOWN | RIDE_LIFECYCLE_DUE_INSPECTION))
-        if (((gCurrentTicks >> 1) & 255) == static_cast<uint32_t>(id))
+    {
+        // Breakdown updates are distributed, only one ride can update the breakdown status per tick.
```

Comment:
But why. That seems like a bad idea. Perhaps if we did `(gCurrentTicks & 255) == (rideId & 255)` then it does not change anything balance wise from the base game but allows for the new limits

Reply:
I think the reason it was built like this is to avoid having news popups overwriting each other. If I wrap the ride id its possible that two or more rides breakdown each adding the news entry in the same tick where normally the distance of breakdowns is delayed by ride id distance.

Reply:
Can we get a resolution for this?

Reply:
I still feel we should do it `(gCurrentTicks & 0xFF) == (rideId & 0xFF)` as is atm rides will breakdown much less frequently.

Reply:
Makes sense